vkcs/data_source_vkcs_publicdns_zone.go

```diff
@@ -0,0 +1,103 @@
+package vkcs
+
+import (
+	"context"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourcePublicDNSZone() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourcePublicDNSZoneRead,
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+
+			"id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The UUID of the DNS zone",
+			},
+
+			"primary_dns": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"admin_email": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"serial": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+
+			"refresh": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+
+			"retry": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+
+			"expire": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+
+			"ttl": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+
+			"zone": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"status": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
```

Comment:
don't we need a description for the datasource?

Reply:
Added: https://github.com/vk-cs/terraform-provider-vkcs/blob/0eb5e7947c41bb51fe453810073b0bb23655bf54/vkcs/data_source_vkcs_publicdns_zone.go#L92